src/core/ext/filters/client_channel/subchannel.h

```diff
@@ -189,6 +189,7 @@ class Subchannel {
   void Unref(GRPC_SUBCHANNEL_REF_EXTRA_ARGS);
   Subchannel* WeakRef(GRPC_SUBCHANNEL_REF_EXTRA_ARGS);
   void WeakUnref(GRPC_SUBCHANNEL_REF_EXTRA_ARGS);
+  // Try to ref from a state where only weak refcount is guaranteed non-zero.
```

Comment:
Let's make the wording a bit clearer.  How about something like this:

Attempts to return a new ref to the subchannel from a weak ref.  If the weak ref count is zero, does not alter the refcount and returns null.